source/manuals/accessibility.html.md.erb

```diff
@@ -36,11 +36,29 @@ This is why [semantic markup](https://html.com/semantic-markup/) and good headin
 
 Accessibility testing tools such as [aXe](https://www.deque.com/axe/) and [Lighthouse](https://developers.google.com/web/tools/lighthouse/) are useful for finding issues, but [automated testing can only find around 30% of likely accessibility problems](https://alphagov.github.io/accessibility-tool-audit/). Automated testing shouldn’t be the only accessibility testing carried out on a service, it should also be tested with real users.
 
-### Browser compatibility testing
+### Testing with assistive technologies
 
-Services should be thoroughly tested in all browsers they are required to be compatible with, including mobile devices and accessibility tools such as screen readers. Emulated browser testing services are available, but some testing on real mobile devices is also recommended.
+Test your service with assistive technologies throughout development, especially when you introduce a significant feature or make any major change. You should test in [the assistive technology and browser combinations listed in the service manual](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies).
 
-For reference, these are the [browsers that GOV.UK is required to be compatible with](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices) and a service should also work with these [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test).
+Read more about [using JAWS](https://webaim.org/articles/jaws/), [using NVDA](https://webaim.org/articles/nvda/) and [using VoiceOver](https://webaim.org/articles/voiceover/).
+
+You can run Voiceover training on macOS by going to System Preferences > Accessibility > VoiceOver > Open VoiceOver Training…
+
+#### Using Assistiv Labs to test with assistive technologies
+
+You can use [Assistiv Labs](https://assistivlabs.com/) to test your service with JAWS and NVDA.
+
+Contact Matt Hobbs (Head of Frontend) to request an account.
```

Comment:
We usually don't name people because the content becomes out-of-date when they move on. Could we just say 'Head of Frontend'?